pom.xml

```diff
@@ -746,8 +746,7 @@
                     </archive>
                     <includes>
                         <include>com/**/*</include>
-                        <include>io/**/*</include>
-                        <include>resources/**/*</include>
+                        <include>resources/properties/default/*</include>
```

Comment:
@MahmoudElSharkawy we use these patterns to define which files should be included in SHAFT's jar. changing it to only the properties directory will remove other sub folders and thus save overall jar file size.